ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/container/clientlibs/js/header.js

```diff
@@ -0,0 +1,10 @@
+document.querySelector(".search i.cmp-search__icon").addEventListener("click", headerSearch);
```

Comment:
@Rajamouli-hcl : file name should be container.js

Reply:
updated to container.js